src/backend/main/py.pi/caliopen_pi/features/types.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+"""Caliopen privacy features definition."""
+
+from __future__ import unicode_literals, absolute_import
+
+import logging
+
+# Privacy features definition
+FEATURES_0 = {'technic': [{'mail_emitter_mx_reputation': {'type': 'string'}},
+                          {'mail_emitter_certificate': {'type': 'string'}},
+                          {'mail_agent': {'type': 'string'}},
+                          {'transport_signed': {'type': 'bool'}},
+                          {'message_signed': {'type': 'bool'}},
+                          {'message_signature_type': {'type': 'string'}},
+                          {'message_encrypted': {'type': 'bool'}},
+                          {'message_encryption_infos': {'type': 'string'}},
+                          {'spam_score': {'max': 100,
+                                          'min': 0,
```

Comment:
yes it's badly normalized, need to rework that part as now I know which headers to consider and values to expect